internal/exec/shell_utils.go

```diff
@@ -154,6 +154,7 @@ func execTerraformShellCommand(
 	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_import=-var-file=%s", varFile))
 	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_destroy=-var-file=%s", varFile))
 	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_console=-var-file=%s", varFile))
+	componentEnvList = append(componentEnvList, "PS1=atmos>")
```

Comment:
Let's use the format of the previous examples for consistency

Reply:
@pkbhowmick see https://github.com/cloudposse/atmos/pull/761#issuecomment-2453567203

Reply:
Something more like:

```
componentEnvList = append(componentEnvList, fmt.Sprintf("PS1=[atmos:%s/%s]> ", stack, component))
```

Reply:
Also, please enhance the error handling per @coderabbitai

```
 		if len(shellCommand) == 0 {
 			bashPath, err := exec.LookPath("bash")
 			if err != nil {
+				// Try fallback to sh if bash is not available
+				shPath, shErr := exec.LookPath("sh")
+				if shErr != nil {
+					return fmt.Errorf("no suitable shell found: %v", err)
+				}
+				shellCommand = shPath
+			} else {
 				return err
+			}
 			}
 			shellCommand = bashPath
 		}
```